### PR TITLE
Enable workflow from pipeline

### PR DESCRIPTION
**Pull request by v-amolpatil**
Required items, please complete
   
   Change(s):
   - Added variable Group, moved code from ps1 file to pipeline. Checkout is need to make use of powershell file so moved the deleted file content to pipeline.

   Reason for Change(s):
   - Azure pipeline should be able to run github workflow.

   Version Updated:
   - NA

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: .azure-pipelines/callGithubWorkflow.yaml [hunk 1]
```diff
@@ -60,7 +62,40 @@ jobs:
...
+              "Accept" = "application/vnd.github+json"
+          }
+
+          $githubToken = "$($env:GITHUB_TOKEN_VALUE)"
+          $token = $githubToken | ConvertTo-SecureString -AsPlainText -Force
+          $pullRequestNumberInt = [int]$pullRequestNumber
+          $client_payload = @{
+              "command" = "ping"
+              "pullRequestBranchName" = "$branchName"
+              "pullRequestNumber" = $pullRequestNumberInt
```
Comment (Tichandr) [inline .azure-pipelines/callGithubWorkflow.yaml:76]: Is there a need to pass the PR number?

File: .azure-pipelines/callGithubWorkflow.yaml [hunk 2]
```diff
@@ -60,7 +62,40 @@ jobs:
         # NOT MAIN/MASTER OR AUTOMATED BRANCH
         if ($branchName -ne 'master' -and $isAutoGeneratedPR -eq $false -and $targetBranch -eq 'master')
         {
-          .script/package-automation/invokeGithubWorklow.ps1 $branchName $pullRequestNumber
+          # INVOKE GITHUB WORKFLOW
+          $header = @{
+              "Accept" = "application/vnd.github+json"
```
Comment (Tichandr) [inline .azure-pipelines/callGithubWorkflow.yaml:67]: what is this header?

File: .azure-pipelines/callGithubWorkflow.yaml (hunk 1 above)
Comment (v-amolpatil) [reply .azure-pipelines/callGithubWorkflow.yaml:76]: yes we need it in our github workflow run

File: .azure-pipelines/callGithubWorkflow.yaml [hunk 3]
```diff
@@ -60,7 +62,40 @@ jobs:
...
+          }
+
+          $githubToken = "$($env:GITHUB_TOKEN_VALUE)"
+          $token = $githubToken | ConvertTo-SecureString -AsPlainText -Force
+          $pullRequestNumberInt = [int]$pullRequestNumber
+          $client_payload = @{
+              "command" = "ping"
```
Comment (Tichandr) [inline .azure-pipelines/callGithubWorkflow.yaml:74]: what is the use of this command ping

File: .azure-pipelines/callGithubWorkflow.yaml (hunk 2 above)
Comment (v-amolpatil) [reply .azure-pipelines/callGithubWorkflow.yaml:67]: we are making api call to github to run workflow for us.

File: .azure-pipelines/callGithubWorkflow.yaml (hunk 3 above)
Comment (v-amolpatil) [reply .azure-pipelines/callGithubWorkflow.yaml:74]: removed this line

**Review by Tichandr — DISMISSED** [review]
Looks good

**Review by Tichandr — APPROVED** [review]
Looks Good

**Pull request merged**